gadgets/Makefile

```diff
@@ -37,6 +37,7 @@ build: $(GADGETS)
 $(GADGETS):
 	@echo "Building $@"
 	@sudo -E IG_EXPERIMENTAL=true $(IG) image build \
+		--update-metadata \
```

Comment:
It does not belong to this PR.

I am fine with adding it, I tried to add it in another PR, but AFAIR @mauriciovasquezbernal preferred to leave it out. Meanwhile, you can use the command from line  35.